src/ring_pedersen.rs

```diff
@@ -0,0 +1,334 @@
+// Copyright (c) 2023 Bolt Labs, Inc.
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree and the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree.
+
+//! Implements a (verified) ring-Pedersen commitment scheme, as specified in Definition 1.3 of
+//! <https://eprint.iacr.org/2021/060.pdf>. The verified variant includes a zero-knowledge proof
+//! that the commitment scheme parameters were constructed correctly.
```

Comment:
nit: Please make the first sentence short so it's easy to scan in the built docs. Like
```rust
//! A verifiable ring-Pedersen commitment scheme.
//!
//! This is a ring variant of the Pedersen commitment scheme, as specified ...